### PR TITLE
Add first part of design for LACP support

### PR DESCRIPTION
**Pull request by rozdy**
Related to #3439

### CODE REVIEW

File: docs/design/lacp/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+ * establish LACP session with aggregators
+ * create flows terminated on aggregated ports
+ 
+ Traffic from any source endpoint of such flows should be balanced over all destination endpoints which has proper LACP session established.
+
+## LACP session control
+
+![LACPDU capture](lacpdu-capture.png "LACPDU capture")
+
+Additional default openflow rule is installed on each switch to catch LACP packets and send them to the controller. This rule has match by eth_type=0x8809 (Slow protocols).
```
Comment (timofei-durakov) [inline docs/design/lacp/README.md:19]: rule should be mettered, worth mentioning

File: docs/design/lacp/README.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+Separate storm topology is used to handle all LACP related events. Main part of aggregation topology is the AggregatorBolt. Aggregator bolt should use in-memory storage to save information about partner systems connected to different ports. Storage structure: switch-port tuple is a key and partner system info is a value. Partner system info includes system priority, system, key, port priority, port and state. Two links can be aggregated correctly if they have the same partner system and key but different port. Open-kilda can aggregate any ports so it should respond with constant system and key but different port. Port value should be copied from incoming LACPDU to avoid port collisions on partner side.
```
Comment (timofei-durakov) [inline docs/design/lacp/README.md:21]: what if topology is restarted, how this lacp sessions should be restored? any side effects on that?

**Review by nikitamarchenko — APPROVED** [review]

File: docs/design/lacp/README.md (hunk 1 above)
Comment (niksv) [reply docs/design/lacp/README.md]: fixed

File: docs/design/lacp/README.md (hunk 2 above)
Comment (niksv) [reply docs/design/lacp/README.md]: topology was removed from the design

**Review by niksv — APPROVED** [review]

**Review by dmitrii-beliakov — APPROVED** [review]

**Pull request merged**